### DISCUSSION TITLE
编译后图片超出页面

### DISCUSSION THREAD

**Discussion by ppppls115** [note286/xduts#174] [Q&A] [answered]
![image](https://github.com/note286/xduts/assets/112673832/23cae3f3-31ea-4569-b17d-0d113e6cd810)
我的图片代码是这样：
\begin{figure}[h]
    \centering
    \includegraphics[width= 13cm]{figures/Figure_3.png}
    \caption{XXXX}
    \label{fig:fg3}
\end{figure}

**Comment by note286** [comment]
如果不能提供复现问题的代码，那么可能将无法得到帮助。

**Comment by Latiest** [comment]
同学你解决了吗，我编译以后文字超出了界面

**Comment by Latiest** [comment]
![image](https://github.com/note286/xduts/assets/40340670/6b4ee967-9220-4b36-9f44-007a6f3a61f3)

**Comment by sikouhjw** [comment]
盲猜都用了 `longtable` 吧

**Reply by Latiest** [reply]
我没有哎，检查了一下

**Reply by Latiest** [reply, marked as answer]
原来是tabularx的问题，又要开始纠结我那么长的表要怎么画了

**Reply by note286** [reply]
tabularray
